qa/tests/test_register_page.py

```diff
@@ -0,0 +1,64 @@
+from qa.PageObjects.LoginPage import LoginPage
+from qa.PageObjects.RegisterPage import RegisterPage
+from qa.utilities.BaseClass import BaseClass
+
+
+class TestHomePage(BaseClass):
+    def test_register_user(self):
+        log = self.getLogger()
+        register_page = RegisterPage(self.driver)
+        self.driver.get("http://localhost:5000/register")
+        log.info("Filling in the register form with new user data")
+        register_page.register_user("NewUser2",
+                                    "NewUser2@gmail.com",
```

Comment:
Move test credentials to one place, e.g. env variables or config.py file with global used variables